README.md

```diff
@@ -1,5 +1,7 @@
 <img src="/Images/Banner.png" />
 
+[![Build Status](https://travis-ci.org/Shopify/FunctionalTableData.svg?branch=travis-support)](https://travis-ci.org/Shopify/FunctionalTableData)
```

Comment:
This should not have the branch specified right?

Reply:
Correct! Will fix